common/cli/app.go

```diff
@@ -71,6 +71,16 @@ func (app *App) SetCustomAppHelpTemplate(appHelperTemplate string) {
 	app.CustomAppHelpTemplate = appHelperTemplate
 }
 
+// SetSubcommandHelpTemplate sets custom SubcommandHelpTemplate to cli
+func SetSubcommandHelpTemplate(subcommandHelpTemplate string) {
+	cli.SubcommandHelpTemplate = subcommandHelpTemplate
+}
+
+// SetCommandHelpTemplate sets custom CommandHelpTemplate to cli
+func SetCommandHelpTemplate(commandHelpTemplate string) {
+	cli.CommandHelpTemplate = commandHelpTemplate
+}
+
```

Comment:
```suggestion
// SetCommandHelpTemplate sets custom HelpTemplate to command
func (cmd *Command) SetCustomHelpTemplate(commandHelpTemplate string) {
	cmd.CustomHelpTemplate = commandHelpTemplate
}
```

Reply:
@levkoburburas I don't think it works. 
Looking at urfave, it should work, but the custom help template is not being displayed when I tried it.
FYI, here you go: https://github.com/pastelnetwork/pastel-utility/blob/misc/setting-up-gonod-cli-tool/cmd/init.go#L19-L21
```
initCommand := cli.NewCommand("init")

initCommand.CustomHelpTemplate = CommandHelpTemplate
Am I missing something?
```

Reply:
@levkoburburas am I missing something?

Reply:
I don't know what you are doing wrong (don't have time to figure out), but from [their code](https://github.com/urfave/cli/blob/45952a7d1b0e956db2099493fe1a42837ff5bd0d/help.go#L198-L201). It must work.

Reply:
@levkoburburas let me debug then, and see where it goes wrong.

Reply:
@levkoburburas did you get a chance to try it [here](https://github.com/pastelnetwork/pastel-utility/blob/misc/setting-up-gonod-cli-tool/cmd/init.go#L19-L21)?

Reply:
@edentech88 I don't have enough time to fix code every PR I review. if something doesn't work, you have to find on your own. But if you are sure that this is the `urfave` issue, not in your code. Let me know I will check everything

Reply:
@levkoburburas it's not `urfave` issue, but rather gonode/common/cli

Reply:
It doesn't matter where `urfave` or `gonode/common/cli`, the main thing is not in your code. 
But if it still turns out that issue is in your code after spending time. I will have to leave negative feedback. I would advise you to double check your code.